app/services/notify_service.rb

```diff
@@ -53,8 +53,13 @@ def direct_message?
     @notification.type == :mention && @notification.target_status.direct_visibility?
   end
 
+  def replied_to_status_is_direct_message?
+    replied_to_status = Status.find(@notification.target_status.in_reply_to_id)
```

Comment:
This whole method is just:

`@notification.target_status.thread&.direct_visibility?`

Reply:
Ah, I was looking for something like that and couldn't find :)
Changed it :)